procto.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |gem|
   gem.extra_rdoc_files = %w[LICENSE README.md CONTRIBUTING.md]
   gem.license          = 'MIT'
 
-  gem.add_development_dependency 'bundler', '~> 1.6.1'
+  gem.add_development_dependency 'bundler', '~> 1.11.2'
```

Comment:
I wonder if, for a small gem like this which uses no particularly interesting bundler features, it would be better to relax the required bundler version to something like `~> 1.11`. This would prevent issues like https://github.com/snusnu/procto/issues/1 from happening every so often.